app/src/DesignSystem/Caption.svelte

```diff
@@ -0,0 +1,13 @@
+<style>
+  p {
+    font-family: "GT America Medium";
```

Comment:
Wouldn't we set the main font face globally for the entire app?

Reply:
No, because the font face includes the weight of the font.